pom.xml

```diff
@@ -62,6 +62,11 @@
             <id>sk89q-snapshots</id>
             <url>http://maven.sk89q.com/artifactory/repo</url>
         </repository>
+        <repository>
+        <id>sk89q-repo</id>
```

Comment:
Can you merge this repo with the repo above it? They sound like they should be more or less the same?